projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -228,106 +239,112 @@
+  "tooltip.wilden_spike": "荒野守卫者的掉落物，此生物在寒冷群系生成。",
   "tooltip.wilden_wing": "荒野伏击者的掉落物",
   "tooltip.wilden_horn": "荒野兽群猎手的掉落物",
   "entity.ars_nouveau.summon_wolf": "被召唤的狼",
   "tooltip.worn_notebook": "新生魔艺的详细资料",
-  "effect.ars_nouveau.shocked": "触电",
+  "effect.ars_nouveau.shocked": "感电",
   "ars_nouveau.flask.charges": "可用次数：%d",
   "item.ars_nouveau.potion_flask": "药水烧瓶",
-  "ars_nouveau.familiar_desc.bookwyrm": "书龙使魔能减少任何法术的魔力消耗，并且无条件增加1点法术伤害。在一只书龙旁进行绑定仪式可获得该使魔。",
-  "ars_nouveau.familiar_desc.drygmy": "德格米使魔能增加2点土元素学派法术的伤害，还能增加杀敌后的掉落物数量。在一只德格米旁进行绑定仪式可获得该使魔。",
-  "ars_nouveau.familiar_desc.starbuncle": "星宝石兽使魔能持续给予你速度II状态。除此之外，对星宝石兽使用金粒会让你短暂得到金矿石探知状态。在一只星宝石兽旁进行绑定仪式可获得该使魔。",
-  "ars_nouveau.familiar_desc.whirlisprig": "风转草使魔能使土元素学派魔符的魔力消耗减半，还能让主人进食时恢复额外的饱和度。在一只风转草旁进行绑定仪式可获得该使魔。",
-  "ars_nouveau.familiar_desc.wixie": "薇克精使魔能增加药水的持续时间。战斗时薇克精还会对敌人使用能造成伤害的药水。在一只薇克精旁进行绑定仪式可获得该使魔。",
+
+  "ars_nouveau.familiar_desc.bookwyrm": "书龙使魔能减少所有法术的魔力消耗，并使其造成的伤害增加1点。在书龙旁进行绑定仪式可获得该使魔。",
+  "ars_nouveau.familiar_desc.drygmy": "德格米使魔能增加2点土元素学派法术的伤害，并增加杀敌后的掉落物数量。在德格米旁进行绑定仪式可获得该使魔。",
+  "ars_nouveau.familiar_desc.starbuncle": "星宝石兽使魔能持续给予你速度II效果。此外，给予它们金粒能让你短时间内能占卜到金矿石的位置。在星宝石兽旁进行绑定仪式可获得该使魔。",
+  "ars_nouveau.familiar_desc.whirlisprig": "风转草使魔能减半土元素学派魔符的魔力消耗，并让主人进食时恢复额外的饱和度。在风转草旁进行绑定仪式可获得该使魔。",
+  "ars_nouveau.familiar_desc.wixie": "薇克精使魔能增加药水的持续时间。战斗时薇克精还会向敌人投掷伤害性药水。在薇克精旁进行绑定仪式可获得该使魔。",
   "ars_nouveau.familiar_name.bookwyrm": "书龙",
   "ars_nouveau.familiar_name.drygmy": "德格米",
   "ars_nouveau.familiar_name.starbuncle": "星宝石兽",
   "ars_nouveau.familiar_name.whirlisprig": "风转草",
   "ars_nouveau.familiar_name.wixie": "薇克精",
   "ars_nouveau.glyph_desc.accelerate": "增加弹射物法术的飞行速度。",
-  "ars_nouveau.glyph_desc.amplify": "可叠加地增加大多数法术效果的强度。可增加破坏的挖掘等级和增加法术的伤害。",
-  "ars_nouveau.glyph_desc.aoe": "法术会影响目标方块附近一个更大的范围。",
-  "ars_nouveau.glyph_desc.aquatic": "造成水下呼吸状态。",
-  "ars_nouveau.glyph_desc.blink": "传送施法者。如果击中了实体，而且施法者副手正持有折越卷轴，则被击中的实体会折越到卷轴上的位置。当对自身使用，施法者会向前闪现。法术炮塔和符文可根据相邻容器内的折越卷轴传送实体，此时不消耗卷轴。",
-  "ars_nouveau.glyph_desc.bounce": "使玩家获得弹性状态，该状态能使玩家在摔落时弹起。强化后能在弹起时保留更多向前的动量。",
-  "ars_nouveau.glyph_desc.break": "你的初始法术构件。可破坏平均硬度的方块。可被强化以增加挖掘等级。敏感构件会使其模拟用剪刀破坏方块，而不是镐。",
-  "ars_nouveau.glyph_desc.cold_snap": "定住已经潮湿或在缓慢状态的实体，并对其造成爆发伤害。附近不是缓慢状态的敌人也会进入缓慢状态。目标附近已是潮湿或缓慢状态的实体也会被寒流击中。可被强化、延长时效和范围效果增强。",
-  "ars_nouveau.glyph_desc.conjure_water": "在目标位置放置水，或者帮着火的实体灭火。",
+  "ars_nouveau.glyph_desc.amplify": "增加大多数法术效果的强度，比如破坏的挖掘等级和法术的伤害，可叠加。",
+  "ars_nouveau.glyph_desc.aoe": "扩大法术命中后产生作用的范围。",
+  "ars_nouveau.glyph_desc.aquatic": "附加水下呼吸效果。",
+  "ars_nouveau.glyph_desc.blink": "传送施法者至法术落点，或对自身使用以向前闪现。副手持折跃卷轴命中实体可以消耗卷轴强制将其折跃。法术炮塔和符文可读取附近容器中的卷轴并生效，不消耗卷轴。",
+  "ars_nouveau.glyph_desc.bounce": "使玩家获得弹性效果，该效果能使玩家在摔落时弹起。强化后能在弹起时保留更多水平动量。",
+  "ars_nouveau.glyph_desc.break": "你的初始法术构件。可破坏平均硬度（铁挖掘等级）的方块。被强化时增加挖掘等级，添加敏感构件时会模拟用剪刀破坏方块。",
+  "ars_nouveau.glyph_desc.cold_snap": "冻结潮湿或缓慢的实体，爆发寒流并造成伤害。寒流爆发波及其他符合条件的实体时将连锁产生，波及不符合条件的实体时则附加缓慢效果。可被强化、延长时效和扩散增强。",

```

`将连锁产生`这么写总感觉缺个谓语，好像截断了
```suggestion
  "ars_nouveau.glyph_desc.cold_snap": "冻结潮湿或缓慢的实体，爆发寒流并造成伤害。寒流爆发波及其他符合条件的实体时将产生连锁效果，波及不符合条件的实体时则附加缓慢效果。可被强化、延长时效和扩散增强。",
```